info/diktat-kotlin-coding-style-guide-en.md

```diff
@@ -1092,6 +1092,7 @@ fun baz() {
   There should be no spaces before `,`, `:` and `;`. The only exceptions for colon are the following:
   - when `:` is used to separate a type and a supertype, including anonimous object (after `object` keyword)
   - when delegating to a superclass constructor or a different constructor of the same class
+  - when colon is used in annotation to specify use-site target (e.g. `@param:JsonProperty`)
```

Comment:
We are saying that here should be a space? But in example (in e.g) we are saying that here should be no spaces...

Reply:
Fair enough, changed.